tests/test_client.py

```diff
@@ -151,6 +169,25 @@ def test_close_posting_list(client, posting_card, posting_list: PostingList, shi
     assert posting_list.closed
 
 
+@pytest.mark.skipif(not correios, reason="API Client support disabled")
+@mock.patch('zeep.client.OperationProxy.__call__')
```

Comment:
why use mock over vcr here?

Reply:
This is a well know error in logs as you can see below, but I couldn't replicate it using VCR in correios test environment. 
```
A PLP não será fechada, o(s) objeto(s) [PR12345678BR] já estão vinculados em outra PLP!
```
And correios' support gave us a feedback about this issue that this problem is caused by timeouts during instability.